src/azure-cli/azure/cli/command_modules/synapse/operations/sqlpoolsensitivitylabel.py

```diff
@@ -0,0 +1,152 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=unused-argument
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.mgmt.synapse.models import SensitivityLabel, SensitivityLabelSource
+from knack.util import CLIError
+
+
+def sqlpool_sensitivity_label_show(
+        client,
+        sql_pool_name,
+        workspace_name,
+        schema_name,
+        table_name,
+        column_name,
+        resource_group_name):
+
+    return client.get(
+        resource_group_name,
+        workspace_name,
+        sql_pool_name,
+        schema_name,
+        table_name,
+        column_name,
+        "current")  # ToDo: need to change to SensitivityLabelSource.current
+
```

Comment:
Just out of curiosity, This ToDo label is meaningful?

Reply:
removed, so sorry for this, this is because there is issue in the old sdk, I use this ToDo as reminder but forget to remove this when the latest sdk is available.

---
In reply to: [533162734](https://github.com/Azure/azure-cli/pull/16095#discussion_r533162734) [](ancestors = 533162734)